packages/framework/attributor/README.md

```diff
@@ -1,4 +1,84 @@
 # @fluid-internal/attributor
 
-This package contains code for framework support for attribution of content.
-It's currently in early development and should not be relied on for any production code; backwards compatibility is not yet guaranteed.
+This package contains definitions and implementations for framework-provided attribution functionality.
+
+## Quickstart
+
+To turn on op-stream based attribution in your container, use `mixinAttributor` to create a `ContainerRuntime` class which supports querying for attribution information.
+
+```typescript
+import { ContainerRuntime } from "@fluidframework/container-runtime";
+
+const ContainerRuntimeWithAttribution = mixinAttributor(ContainerRuntime);
+
+// ...then, in your ContainerRuntime factory use this class:
+class ContainerRuntimeFactory implements IRuntimeFactory {
+    public async instantiateRuntime(context: IContainerContext, existing?: boolean): Promise<IRuntime> {
+        const runtime = await ContainerRuntimeWithAttribution.load(
+            /* same arguments you would have provided to ContainerRuntime.load */
+        );
+        // do whatever setup is necessary with the runtime here
+        return runtime;
+    }
+}
+```
+
+TODO: document any settings you need to turn on to get DDSes to track attribution
+
+This will cause documents created with this container runtime to store associations which attribute document content to timestamp/user information.
+
+Applications can recover this information using APIs on the DDSes they use. For example, the following code snippet illustrates how that works for `SharedString`:
+
+```typescript
+function getAttributionInfo(attributor: IRuntimeAttribution, sharedString: SharedString, pos: number): AttributionInfo {
```

Comment:
Not 100% clear how I'd get this `attributor: IRuntimeAttribution`. Some API call on the container runtime?

Reply:
yeah, I need to clean this up a bit in https://github.com/microsoft/FluidFramework/pull/13319. Right now I'm making the container runtime implement the functionality itself so it ends up requiring casting, but the more idiomatic way I believe is to move some of the stuff implemented in ContainerRuntime into a separate object which implements the interface and the provider interface and stick that on the scope, then users would be able to access it with the normal way they request `FluidObject`s off of `scope` (which works for anyone that has to container runtime as well as data store runtimes, since the scope is passed down).

It's fair feedback to document this better for public-facing users, so I'll leave this comment unresolved for now to remind myself to update it with more clear instructions.

Reply:
documentation calls this out more explicitly now.